### DISCUSSION TITLE
Handle requests to upstream service

### DISCUSSION THREAD

**Discussion by guiac** [Kong/kong#7875] [Help] [answered]
I need to take the ID from the Jwt token and manipulate the request to add this ID in the request header that goes to the upstream service. Is it possible to do this in Kong?

**Comment by jschmid1** [comment, marked as answer]
Not natively. There are plugins that allow you to write your own functions for various phases though. See here -> https://docs.konghq.com/hub/kong-inc/serverless-functions/
